ultraLatency.py

```diff
@@ -198,6 +198,7 @@
         oh.setTriggerSource(ohboard,options.gtx,0x5) # GBT, 0x0 for GTX
         pass
 
+    oh.setTriggerThrottle(ohboard, options.gtx, options.throttle)
```

Comment:
Default is 0. In principle i agree with Cameron, but it might be beneficial to always write the default so we don,t run with unexpected leftover value